assets/js/app.js

```diff
@@ -188,17 +188,30 @@ app.controller('AboutCtrl', ['$scope', '$rootScope', function($scope, $rootScope
   })
 }])
 
-app.controller('LoginCtrl', ['$scope', '$rootScope', function($scope, $rootScope) {
+app.controller('LoginCtrl', function($scope, $rootScope, $http, $localStorage, Consts) {
   $rootScope.root = {
     route: 'Login',  // this corresponds to the menu item that should be active
     title: 'Login | GitSubmit'
   }
 
   $scope.submit = function() {
-        //TODO:Add code to handle the fields and send to approriate destination
-        alert($scope.username + ' ' + $scope.password)
+    // alert($scope.username + ' ' + $scope.password)
+    $http({
```

Comment:
Yup this commit was before I figured out jquery ajax for cross origin requests. I'll update these.